libtcmu_log.c

```diff
@@ -365,3 +392,146 @@ static struct log_buf *tcmu_log_initialize(void)
 	pthread_mutex_unlock(&g_mutex);
 	return logbuf;
 }
+
+static int append_output(log_output_func f, log_close_func c, void *data,
+				int priority, int dest, const char *name)
```

Comment:
Match the coding style we normally use where you use tabs then spaces to align with the "(".

Check the rest of the patch for this. I will not blow up your email with a comment for each instance :)